core/tests/protractor_desktop/explorationHistoryTab.js

```diff
@@ -92,24 +94,24 @@ describe('Exploration history', function() {
 
     var VERSION_1_STATE_1_CONTENTS = {
```

Comment:
Can you please explain the changes in this a bit?

Reply:
Earlier, the yaml conversion was done from the backend but I have introduced the frontend alternative. Due to this, the relative ordering of the different properties got changed. Hence, I had to reorder these properties for the tests to pass.